docs/README.md

```diff
@@ -18,18 +18,17 @@
 </p>
 
 
-deck.gl is designed to simplify high-performance, WebGL-based visualization of large data sets. Users can quickly get impressive visual results with minimal effort by composing existing layers, or leverage deck.gl's extensible architecture to address custom needs.
+deck.gl is designed to simplify high-performance, WebGL-based visualization of large data sets, both geospatial and non-geospatial. Users can quickly get impressive visual results with minimal effort by composing existing layers, or leverage deck.gl's extensible architecture to address custom needs.
 
-deck.gl maps **data** (usually an array of JSON objects) into a stack of visual **layers** - e.g. icons, polygons, texts; and look at them with **views**: e.g. map, first-person, orthographic.
 
 deck.gl handles a number of challenges out of the box:
 
 * Performant rendering and updating of large data sets
 * Interactive event handling such as picking, highlighting and filtering
-* Cartographic projections and integration with major basemap providers
+* For geospatial data sets, cartographic projections and integration with major basemap providers including Mapbox, Google Maps and ESRI.
 * A catalog of proven, well-tested layers
 
-Deck.gl is designed to be highly customizable. All layers come with flexible APIs to allow programmatic control of each aspect of the rendering. All core classes such are easily extendable by the users to address custom use cases.
+Deck.gl is designed to be *highly customizable*. All layers come with flexible APIs to allow programmatic control of each aspect of the rendering. All core classes such are easily extendable by the users to address custom use cases.
```

Comment:
We typically spell `deck.gl` with lowercase even at beginning of sentences. Since you are editing...